config/application.yml.default

```diff
@@ -34,213 +34,213 @@ acuant_passlive_url: ''
-acuant_upload_image_timeout: '1'
-acuant_get_results_timeout: '1'
-acuant_create_document_timeout: '1'
-add_email_link_valid_for_hours: '24'
+acuant_timeout: 45.0
+acuant_upload_image_timeout: 1.0
+acuant_get_results_timeout: 1.0
+acuant_create_document_timeout: 1.0
+add_email_link_valid_for_hours: 24
 asset_host: ''
-async_wait_timeout_seconds: '60'
-async_stale_job_timeout_seconds: '300'
-aws_http_timeout: '5'
-aws_http_retry_limit: '2'
-aws_http_retry_max_delay: '1'
+async_wait_timeout_seconds: 60
+async_stale_job_timeout_seconds: 300
+aws_http_timeout: 5
+aws_http_retry_limit: 2
+aws_http_retry_max_delay: 1
 aws_kms_key_id: alias/login-dot-gov-test-keymaker
 aws_logo_bucket: ''
 aws_region: 'us-west-2'
-aws_kms_multi_region_enabled: 'false'
+aws_kms_multi_region_enabled: false
 backup_code_cost: '2000$8$1$'
 country_phone_number_overrides: '{}'
-doc_auth_error_dpi_threshold: '290'
-doc_auth_error_sharpness_threshold: '40'
-doc_auth_error_glare_threshold: '40'
-database_pool_idp: '5'
-database_statement_timeout: '2500'
-database_timeout: '5000'
-deliver_mail_async: 'false'
+doc_auth_error_dpi_threshold: 290
+doc_auth_error_sharpness_threshold: 40
+doc_auth_error_glare_threshold: 40
+database_pool_idp: 5
+database_statement_timeout: 2500
+database_timeout: 5000
+deliver_mail_async: false
 deleted_user_accounts_report_configs: '[]'
-disable_csp_unsafe_inline: 'true'
-disable_email_sending: 'true'
-disallow_all_web_crawlers: 'true'
+disable_csp_unsafe_inline: false
+disable_email_sending: false
+disallow_all_web_crawlers: false
 disposable_email_services: '[]'
-doc_auth_attempt_window_in_minutes: '360'
-doc_auth_extend_timeout_by_minutes: '40'
-doc_capture_polling_enabled: 'true'
-doc_auth_client_glare_threshold: '50'
-doc_auth_client_sharpness_threshold: '50'
-doc_auth_enable_presigned_s3_urls: 'false'
-doc_auth_s3_request_timeout: '5'
-doc_auth_error_dpi_threshold: '290'
-doc_auth_error_glare_threshold: '40'
-doc_auth_error_sharpness_threshold: '40'
-doc_auth_max_attempts: '20'
-doc_auth_max_capture_attempts_before_tips: '3'
-doc_capture_request_valid_for_minutes: '15'
+doc_auth_attempt_window_in_minutes: 360
+doc_auth_extend_timeout_by_minutes: 40
+doc_capture_polling_enabled: false
+doc_auth_client_glare_threshold: 50
+doc_auth_client_sharpness_threshold: 50
+doc_auth_enable_presigned_s3_urls: false
+doc_auth_s3_request_timeout: 5
+doc_auth_error_dpi_threshold: 290
+doc_auth_error_glare_threshold: 40
+doc_auth_error_sharpness_threshold: 40
+doc_auth_max_attempts: 20
+doc_auth_max_capture_attempts_before_tips: 3
+doc_capture_request_valid_for_minutes: 15
 email_from: no-reply@login.gov
 email_from_display_name: Login.gov
-enable_load_testing_mode: 'false'
-enable_partner_api: 'false'
-enable_rate_limiting: 'true'
-enable_test_routes: 'true'
-enable_usps_verification: 'true'
-event_disavowal_expiration_hours: '240'
+enable_load_testing_mode: false
+enable_partner_api: false
+enable_rate_limiting: false
+enable_test_routes: false
```

Comment:
this needs to be `true` ... it's causing tests to fail. Is it possible the boolean parsing is not preserving the right values?

Reply:
This is fixed.